### PR TITLE
Fixes date change failing log ftp

### PR DESCRIPTION
**Pull request by Jordie0608**
world.realtime returns current date and thus invalid path if date changed during round.
Also changed the failure message to be less codery, not that it'll ever be seen now.

### CODE REVIEW

File: code/modules/admin/verbs/getlogs.dm [hunk 1]
```diff
@@ -85,11 +85,10 @@
 	set name = "Show Server Log"
 	set desc = "Shows today's server log."
 
-	var/path = "data/logs/[time2text(world.realtime,"YYYY/MM-Month/DD-Day")].log"
-	if( fexists(path) )
-		src << ftp( file(path) )
+	if(fexists("[diary]"))
+		src << ftp(file("[diary]"))
 	else
```
Comment (MrStonedOne) [inline code/modules/admin/verbs/getlogs.dm]: protip, this is already a /file, so you could just do ftp(diary)

Comment (Jordie0608) [inline code/modules/admin/verbs/getlogs.dm]: I tried that first and it didn't work, I might've done it wrong but ¯\\_(ツ)_/¯

Comment (MrStonedOne) [inline code/modules/admin/verbs/getlogs.dm]: That's not surprising... byond and files can be odd.

Comment (Jordie0608) [inline code/modules/admin/verbs/getlogs.dm]: Nevermind I am a wizard

**Pull request merged**